### PR TITLE
Add Zion Trail

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
Not finalized, see #351 for more information.

### CODE REVIEW

**Comment by pStyl3** [comment]
`&flagESM` has been added to the prelude, check issue 351.

File: masterlist.yaml [hunk 1]
```diff
@@ -2636,6 +2636,27 @@ plugins:
...
+  - name: 'ZionTrail(-NV)?\.es[mp]'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/61939/'
+        name: 'Zion Trail'
+  - name: 'ZionTrail.esm'
+    msg:
+      - <<: *obsolete
+        condition: 'checksum("ZionTrail.esm", 914129FC)'
+    clean:
+    # 1.11
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
```

File: masterlist.yaml [hunk 2]
```diff
@@ -2636,6 +2636,27 @@ plugins:
...
+      - crc: 0xF7CF87B4
+        util: 'FNVEdit v4.0.4c'
+  - name: 'ZionTrail-NV.esp'
+    msg:
+      - <<: *obsolete
+        condition: 'checksum("ZionTrail-NV.esp", B04CEAA2)'
+    clean:
+    # 1.11
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
```

File: masterlist.yaml [hunk 3]
```diff
@@ -2636,6 +2636,25 @@ plugins:
...
+        name: 'Zion Trail'
+  - name: 'ZionTrail.esm'
+    msg:
+      - <<: *obsolete
+        condition: 'checksum("ZionTrail.esm", 914129FC)'
+    clean:
+      - crc: 0xF7CF87B4
+        util: 'FNVEdit v4.0.4c'
+  - name: 'ZionTrail-NV.esp'
+    msg:
```
Comment (MacSplody) [inline masterlist.yaml:2908]:
```suggestion
    msg:
      - <<: *flagESM
        condition: 'not is_master("ZionTrail-NV.esp")'
```

**Review by MacSplody — APPROVED** [review]

**Pull request merged**